pytorch_lightning/callbacks/quantization.py

```diff
@@ -81,6 +82,90 @@ def wrapper(data) -> Any:
     return wrapper
 
 
+def _is_quantize_forward_context_wrapped(model):
+    wrapping = wrap_quantize_forward_context
+    fn = inspect.getsourcefile(wrapping)
+    lines, first_line = inspect.getsourcelines(wrapping)
+    last_line = first_line + len(lines) - 1
+
+    fw_method = model.forward
+    while True:
+        if first_line <= fw_method.__code__.co_firstlineno <= last_line and fw_method.__code__.co_filename == fn:
+            return True
+        fw_method = getattr(fw_method, "__wrapped__", None)
+        if fw_method is None:
+            return False
+
+
+def _fixup_quantization_torchscript(script_module):
```

Comment:
noob question, but do you believe this utility could have its place in PyTorch ?